modules/vulnerability-management-migration.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_mod-docs-content-type: CONCEPT
+[id="vulnerability-management-migration_{context}"]
+= Migration of vulnerability reports when upgrading to RHACS version 4.3 and later
+
+[role="_abstract"]
+{product-title-short} version 4.3 includes an automatic migration of vulnerability report configurations that were created in previous versions of {product-title-short} under *Vulnerability Management 1.0* -> *Reporting*. Migrated report configurations are moved under *Vulnerability Management (2.0)*, and the previous versions of the report configurations are no longer available in the web portal or by using the API.
```

Comment:
It's the ACS GUI, which is different than the OCP web console. Our docs call it the web portal so that is a familiar term to ACS users. Vulnerability Management -> Reporting is a nav menu item.

Reply:
(Will change to refer to it as "RHACS web portal" for first use)